src/main/java/de/tum/in/www1/artemis/domain/metis/conversation/UserConversationInfo.java

```diff
@@ -15,10 +15,10 @@ public class UserConversationInfo {
 
     private final long unreadMessagesCount;
 
-    public UserConversationInfo(Long conversationId, Long participantId, Boolean isModerator, Boolean isFavorite, Boolean isHidden, ZonedDateTime lastRead,
+    public UserConversationInfo(Long conversationId, Long participantId, Boolean isModerator, Boolean isFavorite, Boolean isHidden, Boolean isMuted, ZonedDateTime lastRead,
```

Comment:
Should `isMuted` be nullable here? I'd default to false if it is not specifically set.

Reply:
In general all of these types could probably be primitives instead of wrappers 🤷🏻 

Might be something for a follow up PR.

Reply:
Will make isMuted non-nullable :)

Reply:
Honestly this entire Boolean handling seems a little funky to me, so it is fine if you leave it as is. Though maybe you can note this down as an issue or something for future developers.